.env

```
@@ -0,0 +1,5 @@
+# Supabase Configuration
+# Get these from: https://app.supabase.com/project/_/settings/api
+NEXT_PUBLIC_SUPABASE_URL=https://xoushfbwvicjwnippuic.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InhvdXNoZmJ3dmljanduaXBwdWljIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjIyNjI0OTcsImV4cCI6MjA3NzgzODQ5N30.1RoVern_FC1bbtoHvfPqQuivEzp0_y-jXA4EEW9eNRQ
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)